_security-analytics/api-tools/detector-api.md

```diff
@@ -308,72 +331,103 @@ PUT /_plugins/_security_analytics/detectors/J1RX1IMByX0LvTiGTddR
 }
 ```
 
+#### Response fields
+
+Field | Type | Description
+:--- | :--- |:--- |:--- |
+`_version` | String | Version number for the update.
+`detector.last_update_time` | String | Date and time of the last update.
+`detector.enabled_time` | String | Date and time when the detector was last enabled.
+
 ---
-## Delete Detector
+## Delete Detector API
 
-This API is used for deleting a detector.
+This API uses the detector ID to specify and delete a detector.
 
 ### Sample request
 
 ```json
-DELETE /_plugins/_security_analytics/detectors/J1RX1IMByX0LvTiGTddR
+DELETE /_plugins/_security_analytics/detectors/<detector Id>
+```
+{% include copy-curl.html %}
+
+### Sample response
+
+```json
+{
+  "_id" : "IJAXz4QBrmVplM4JYxx_",
+  "_version" : 1
+}
 ```
 
 ---
-## Get Detector
+## Get Detector API
 
-The Get detector API retrieves the detector details.
+The Get Detector API retrieves the detector details. Use the detector ID in the call to fetch detector details.
 
 ### Sample request
 
 ```json
-GET /_plugins/_security_analytics/detectors/MFRg1IMByX0LvTiGHtcN
+GET /_plugins/_security_analytics/detectors/<detector Id>
```

Comment:
I would create a "Path and HTTP methods" section before sample request, where I would have <detector ID>, and here I would provide an actual detector ID because it's nice to see an example.

Reply:
Added (for DELETE, as well). But will ask you about this later.